From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Pekka Vuorela <pekka.vuorela@jolla.com>
Date: Thu, 29 Sep 2022 12:07:37 +0300
Subject: [PATCH] autogen: Do not fetch sources at the build time

Use git submodule and copy gnulib sources instead of
fetching build time.

Signed-off-by: Matti Kosola <matti.kosola@jolla.com>
---
 autogen.sh | 12 +-----------
 1 file changed, 1 insertion(+), 11 deletions(-)

diff --git a/autogen.sh b/autogen.sh
index d66f3cd6..49329bb1 100755
--- a/autogen.sh
+++ b/autogen.sh
@@ -28,19 +28,9 @@ modules="
 	wctype-h
 	wcwidth
 "
-
-# Make sure the local gnulib git repo is up-to-date.
 if [ ! -d "gnulib" ]; then
-	git clone --depth=1111 ${gnulib_url}
-fi
-cd gnulib >/dev/null || exit 1
-curr_hash=$(git log -1 --format=%H)
-if [ "${gnulib_hash}" != "${curr_hash}" ]; then
-	echo "Pulling..."
-	git pull
-	git checkout --force ${gnulib_hash}
+       cp -lr ../gnulib .
 fi
-cd .. >/dev/null || exit 1
 
 echo "Autopoint..."
 autopoint --force
